discovery/kubernetes/kubernetes.go

```diff
@@ -48,8 +51,96 @@ var (
 		},
 		[]string{"role", "event"},
 	)
+	// DefaultSDConfig is the default Kubernetes SD configuration
+	DefaultSDConfig = SDConfig{}
+)
+
+// KubernetesRole is role of the service in Kubernetes.
+type KubernetesRole string
+
+// The valid options for KubernetesRole.
+const (
+	KubernetesRoleNode     KubernetesRole = "node"
+	KubernetesRolePod      KubernetesRole = "pod"
+	KubernetesRoleService  KubernetesRole = "service"
+	KubernetesRoleEndpoint KubernetesRole = "endpoints"
+	KubernetesRoleIngress  KubernetesRole = "ingress"
```

Comment:
I think all these can lose their `Kubernetes` prefix, since they are inside the `kubernetes` package now and the names are sufficiently clear without the prefix.